wagtail_footnotes/blocks.py

```diff
@@ -20,7 +22,7 @@ def replace_footnote_tags(self, html, context=None):
         def replace_tag(match):
             try:
                 index = self.process_footnote(match.group(1), context["page"])
-            except (KeyError, ValidationError):
+            except (KeyError, ValidationError, Footnote.DoesNotExist):
```

Comment:
I am not sure how enforcing uniqueness relates to a note not existing?

Reply:
Apologies, this is a slightly unrelated change that is still needed, this fixes the error mentioned https://github.com/torchbox/wagtail-footnotes/issues/11 where a footnote referenced might not exist against the page. This can happen when copying text from one page to another.

Reply:
Since I have now merged: https://github.com/torchbox/wagtail-footnotes/pull/2
this is no longer needed